.github/workflows/deploy-pr.yml

```diff
@@ -0,0 +1,29 @@
+name: deploy website to netlify
+
+on: [push, pull_request]
+
+jobs:
+  build-static-page:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: build image
+        run: docker build -t documentation:latest .
+      - name: copy static files locally
+        run: docker run -v ${PWD}:/output documentation:latest cp -r /usr/share/nginx/html /output/_site
+      - name: deploy to netlify
+        uses: ./.github/actions/netlify-deploy
+        id: netlify_deploy
+        with:
+          NETLIFY_ACCOUNT_SLUG: ${{ secrets.NETLIFY_ACCOUNT_SLUG }}
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_DIRECTORY: _site
+          NETLIFY_SITE_NAME: "${{ github.repository }}/${{ github.head_ref }}"
+      - name: notify website is live
+        uses: ./.github/actions/create-status-check
+        with:
+          SITE_URL: ${{ steps.netlify_deploy.outputs.site_url }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          GITHUB_SHA: ${{ github.sha }}
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
nit: missing newline at end of file 😅 (can sometimes be problematic when using git)

Reply:
I see some other files are also having this btw

Reply:
fixed!